liquidctl/driver/corsair_hid_psu.py

```diff
@@ -278,7 +296,7 @@ def quadratic(params, x):
         _LOGGER.debug('input power estimates: %.3f W @ 115 V; %.3f W @ 230 V', for_in115v, for_in230v)
 
         # linearly interpolate for input_voltage
-        return for_in115v + (for_in230v - for_in115v) / 115 * (input_voltage - 115)
+        return for_in115v + (for_in230v - for_in115v) * (input_voltage - 115) / 115
```

Comment:
I had to double check if this was `a / (b * c)` or `(a / b) * c` so I think this way round is clearer / less ambiguous